src/app/pages/queue/question/question.component.ts

```diff
@@ -28,19 +33,19 @@ export class QuestionComponent {
 
   deleteQuestion(id: string) {
     // this.questionService.deleteQuestion(id);
-    console.log('deleting question');
+    this.showToast('Deleted Question', !!this.question.title ? this.question.title : undefined);
```

Comment:
```suggestion
    this.showToast('Deleted Question', this.question.title || null);
```

Reply:
I was considering this...

I was thinking that i would need to specify undefined to make sure it doesn't fill in the optional parameter with null.